pontoon/base/models.py

```diff
@@ -152,6 +153,15 @@ class Locale(models.Model):
 
     team_description = models.TextField(blank=True)
 
+    #: Most recent translation approved or created for this locale.
+    latest_translation = models.ForeignKey(
+        'Translation',
+        blank=True,
+        null=True,
+        related_name='+',
+        on_delete=models.SET_NULL
```

Comment:
It's the other way around: Deleting a translation deletes the locale with the foreign key. This is how I mysteriously deleted projects on my local instance while running a sync, which deleted some translations. 

Suffice it to say that we do _not_ want locales being deleted when translations are deleted. :P